pulsar-testclient/src/main/java/org/apache/pulsar/testclient/PerformanceConsumer.java

```diff
@@ -293,6 +297,11 @@ public static void main(String[] args) throws Exception {
                     PerfClientUtils.exit(0);
                 }
             }
+            if (arguments.numMessages > 0 && totalMessagesReceived.sum() >= arguments.numMessages) {
+                log.info("------------------- DONE -----------------------");
```

Comment:
Could we add a hint that why the perf tool is done? 

Such as `log.inf("------------------- DONE (reach consume message num in total) -----------------------")

Reply:
Good suggestion, I have added hints to all the places where the number of messages and the test duration are reached.